Nasal/c172p.nas

```diff
@@ -191,6 +191,68 @@ var return_fuel_sample = func(index) {
 };
 
 ##########################################
+# Preflight control surface check: left aileron
+##########################################
+var control_surface_check_left_aileron = func {
+    var auto_coordination = getprop("/controls/flight/auto-coordination");
+    setprop("/controls/flight/auto-coordination", "false");
```

Comment:
AFAIK `"false"` is interpreted by `setprop()` as just a string. Usually you would write `0` instead (since Nasal doesn't support the `true`/`false` literals).